frontend/packages/citizen-frontend/src/applications/editor/unit-preference/SiblingBasisSubSection.tsx

```diff
@@ -40,7 +51,12 @@ export default React.memo(function SiblingBasisSubSection({
             <>
               <Gap />
               <Label>
-                {t.applications.editor.unitPreference.siblingBasis.radioLabel} *
+                {
+                  t.applications.editor.unitPreference.siblingBasis.radioLabel[
+                    applicationType
+                  ]
+                }{' '}
+                *
```

Comment:
Supernit: why not include the asterisk in the string:
```suggestion
                }{' *'}
```